app/src/main/java/edu/mills/cs115/HomeFragment.java

```diff
@@ -0,0 +1,185 @@
+  private int[] lists = new int[] {
+      R.drawable.cs1, R.drawable.cs2, R.drawable.cs3, R.drawable.cs4, R.drawable.cs5,
+      R.drawable.cs6, R.drawable.cs7, R.drawable.cs8, R.drawable.cs9, R.drawable.cs10,
+      R.drawable.cs11, R.drawable.cs12, R.drawable.cs13
+  };
+  private List<Integer> curLists = new ArrayList<>();
+  //Refresh button for refresh content of "word of the day"
+  private ImageView refresh;
+  private SQLiteDatabase db;
+  private TextView title_tv;
+  //Add definition for word of "word of the day"
+
+  private WebView webView;
+
+  @Override public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+  }
+
+  @Override public View initView() {
+    View view = View.inflate(mContext, R.layout.fragment_home, null);
+    banner = (ConvenientBanner) view.findViewById(R.id.convenientBanner);
+    title_tv = (TextView) view.findViewById(R.id.title_tv);
+    refresh = (ImageView) view.findViewById(R.id.refresh_iv);
+    webView = (WebView) view.findViewById(R.id.webview);
+    WebSettings mWebSettings = webView.getSettings();
+    mWebSettings.setSupportZoom(true);
+    mWebSettings.setLoadWithOverviewMode(true);
+    mWebSettings.setUseWideViewPort(true);
+    mWebSettings.setDefaultTextEncodingName("utf-8");
+    mWebSettings.setRenderPriority(WebSettings.RenderPriority.HIGH);//。
+    mWebSettings.setJavaScriptEnabled(true);
+    mWebSettings.setSupportMultipleWindows(true);
+
+
+    //Repeat onCreateWindow method in WebChromeClient
+    mWebSettings.setSupportMultipleWindows(false);
+    mWebSettings.setJavaScriptCanOpenWindowsAutomatically(true);
+    refresh.setOnClickListener(new View.OnClickListener() {
+      @Override public void onClick(View v) {
+        setDictionary();
+      }
+    });
+    creator = new CBViewHolderCreator<NetworkImageHolderView>() {
+      @Override public NetworkImageHolderView createHolder() {
+        return new NetworkImageHolderView();
+      }
+    };
+    //Random generate 4 memes pictures for banner
+    for (int i = 0; i < lists.length; i++) {
+      int random = (int) (Math.random() * lists.length);
+      if (curLists.size() < 4) {
+        if (!curLists.contains(lists[random])) {
+          curLists.add(lists[random]);
+        }
+      }
+    }
+    banner.setPages(creator, curLists).setOnItemClickListener(new OnItemClickListener() {
+      @Override public void onItemClick(int position) {
+
+      }
+    }).setPageIndicator(new int[] { R.drawable.banner_unselect, R.drawable.banner_select });
+
+    return view;
+  }
+
+  @Override public void onStart() {
+    super.onStart();
+    setDictionary();
+    banner.startTurning(5000);
+  }
+
+  @Override public void onStop() {
+    super.onStop();
+    if (banner.isTurning()) {
+      banner.stopTurning();
+    }
+  }
+  //Random generate a word  as word of the day content
+  public void setDictionary() {
+    DictionaryBean db = queryRandom();
+    String colorText = "<h1>Word of the Day:</h1>"
```

Comment:
In addition to hardcoding your HTML, you're hardcoding "Word of the Day", which will make translation hard.